cypress/integration-qe/hostStatusInsufficient.spec.js

```diff
@@ -0,0 +1,55 @@
+import { CLUSTER_NAME, openCluster } from './shared';
+
+// @OCP-34205
+//Following prerequisite steps needs to be done before running this test
+// - only 1 VM running
+// - VM CPU is set to 1 vcpu
+const nodeName = 'master-0-0';
+
+describe('Host status insufficient in Cluster Events', () => {
+  before(function () {
+    openCluster(CLUSTER_NAME);
+  });
+
+  it('Should click the View Cluster Events button', () => {
+    cy.get('button[id=cluster-events-button]').click();
+  });
+
+  it('Should open the Cluster Events popup', () => {
+    cy.get('#pf-modal-part-8');
```

Comment:
No need to test the structure of the dialog, it's already covered in clusterDetails.spec.js